synapse/cortex.py

```diff
@@ -1422,6 +1422,10 @@ def getStormSvc(self, name):
         if ssvc is not None:
             return ssvc
 
+        for ssvc in self.svcsbyiden.values():
```

Comment:
This feels like we should have a svcbysvcname dict so we can look these up without iterating.  Though most use cases should be optimal, i could easily see this ending up in a per-node loop